docker-compose.circle.yml

```diff
@@ -94,31 +94,31 @@ centos7build:
 ## Package testing nodes
 #
 wheezytest:
-  image: quay.io/dennybaa/droneunit:wheezy-sshd
+  image: stackstorm/packagingtest:wheezy
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 jessietest:
-  image: quay.io/dennybaa/droneunit:jessie-sshd
+  image: stackstorm/packagingtest:jessie
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 trustytest:
-  image: quay.io/dennybaa/droneunit:trusty-upstart
+  image: stackstorm/packagingtest:trusty
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos6test:
-  image: quay.io/dennybaa/droneunit:centos6-sshd
+  image: stackstorm/packagingtest:centos6
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos7test:
-  image: quay.io/dennybaa/droneunit:centos7-systemd
+  image: stackstorm/packagingtest:centos7
```

Comment:
It would be much helpful to include daemon naming explicitly, eg `-systemd`, `-upstart` is pretty useful.
Dockerized `centos7-systemd` or `trusty-upstart` is not something that you can see every day.